sources/README.md

```diff
@@ -0,0 +1,2 @@
+This is a temporary workaround for twoliter who currently expects a sources
+directory to exist in order for the docker volume mounts to work in buildkit
```

Comment:
Is there a backlog issue in Twoliter to avoid needing this?

Reply:
I'll be adding it, we had a regression on a bug fix I did.